test/fixtures/schema/boolean-literal.json

```diff
@@ -164,7 +164,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"street\": [\n    true,\n    false\n  ]\n}"
+                              "content": "{\n  \"street\": [\n    false,\n    true,\n    false\n  ]\n}"
```

Comment:
Another array type-hint being rendered.

Reply:
Again, this is not a bug in this branch. It's a bug in master. Should I fix it here?

Reply:
The diff is coming from master > this branch. So the bug would be introduced in this branch?

Reply:
The bug surfaces in this branch, because my implementation doesn't try and work around bugs in the API Elements tree. It expects API Elements according to the spec and interprets it accordingly.

The former implementation worked around some of these bugs (i.e. it interpreted too much), and because of that we had wrong API Elements and correct JSON Value.

I can fix the whole type-hint thing in this branch, but I'm not sure how trivial the solution will be (haven't seen the AST -> API Elements transformation code for a while).